chef-config/lib/chef-config/version.rb

```diff
@@ -14,9 +14,7 @@
 # limitations under the License.
 
 #!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
-# NOTE: This file is generated by running `rake version` in the top level of
-# this repo. Do not edit this manually. Edit the VERSION file and run the rake
-# task instead.
+# NOTE: This file is modified automatically by Expeditor. Do not modify it manually.
```

Comment:
it is probably clearer to reference `update-version.sh` here, so it's clear what mechanism causes this update.